smtp/address.ts

```diff
@@ -146,16 +146,19 @@ function convertAddressTokens(tokens: AddressToken[]) {
 			// still no address
 			if (addresses.length === 0) {
 				for (let i = texts.length - 1; i >= 0; i--) {
-					texts[i] = texts[i]
-						.replace(/\s*\b[^@\s]+@[^@\s]+\b\s*/, (address: string) => {
-							if (addresses.length === 0) {
-								addresses = [address.trim()];
-								return ' ';
-							} else {
-								return address;
-							}
-						})
-						.trim();
+					const text = texts[i];
+					texts[i] = text
```

Comment:
i think an `if` / `else` would be easier to read

Reply:
done 19fd8338f5327471531f93cf6575df573d35656c